docs/api.yaml

```diff
@@ -0,0 +1,844 @@
+openapi: "3.0.3"
+info:
+  #  description:
+
+  version: "1.0.0"
+  title: "HEI Admin API"
+  contact:
+    email: "contact@hei.school"
+
+  #servers:
+  # - description: To complete
+  #   url: example.com
+
+paths:
+  /grades:
```

Comment:
Not needed. A grade should be an attribute of a student.enrolled_course. When not yet graded, student.enrolled_course.grade should be null. When giving grade, just patch that attribute

Reply:
Adding a grade is now a PATCH-ed action in student endpoint.